qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/project/QuteProjectRegistry.java

```diff
@@ -198,11 +210,25 @@ public QuteProject getProject(String projectUri) {
 	 * @return the Qute project by the given info <code>projectInfo</code>.
 	 */
 	public QuteProject getProject(ProjectInfo projectInfo) {
+		return getProject(projectInfo, true);
+	}
+
+	/**
+	 * Returns the Qute project by the given info <code>projectInfo</code>.
+	 *
+	 * @param projectInfo    the project information.
+	 * @param validateOnLoad true if validation of templates must be validated if

```

Comment:
If I understand correctly, this is:

`true if validation of Qute templates must be done when project is loaded and false otherwise.`